protocol-designer/src/components/modules/StagingAreasModal.tsx

```diff
@@ -44,17 +44,19 @@ const StagingAreasModalComponent = (
   const { onCloseClick, stagingAreas } = props
   const { values, setFieldValue } = useFormikContext<StagingAreasValues>()
   const initialDeckSetup = useSelector(getInitialDeckSetup)
-  const areSlotsEmpty = values.selectedSlots.map(slot =>
-    getSlotIsEmpty(initialDeckSetup, slot)
-  )
   const hasWasteChute =
     Object.values(initialDeckSetup.additionalEquipmentOnDeck).find(
       aE => aE.name === 'wasteChute'
     ) != null
-  const hasConflictedSlot =
-    hasWasteChute && values.selectedSlots.find(slot => slot === 'cutoutD3')
-      ? false
-      : areSlotsEmpty.includes(false)
+  const areSlotsEmpty = values.selectedSlots.map(slot => {
```

Comment:
nit
maybe using singular?

```suggestion
  const areSlotEmpty = values.selectedSlots.map(slot => {
```